### ISSUE TITLE
alembic.util.exc.CommandError: Can't locate revision identified by '5b0bd3b41a23'

### ISSUE THREAD

**Issue by medphisiker** [PrefectHQ/prefect#10778]
### First check

- [X] I added a descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Prefect documentation for this issue.
- [X] I checked that this issue is related to Prefect and not one of its dependencies.

### Bug summary

Hello,

I have installed environment by poetry with `python = "3.10"` and `prefect = "2.10.21"`.
Everything worked fine for a while. I have not changed my working environment, updated or installed anything.

But recently when I running the standard command to run perfect server:

```
prefect server start
```

I got this response with error:

```

 ___ ___ ___ ___ ___ ___ _____ 
| _ \ _ \ __| __| __/ __|_   _| 
|  _/   / _|| _|| _| (__  | |  
|_| |_|_\___|_| |___\___| |_|  

Configure Prefect to communicate with the server with:

    prefect config set PREFECT_API_URL=http://127.0.0.1:4200/api

View the API reference documentation at http://127.0.0.1:4200/docs

Check out the dashboard at http://127.0.0.1:4200



Traceback (most recent call last):
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/base.py", line 246, in _catch_revision_errors
    yield
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/base.py", line 456, in _upgrade_revs
    for script in reversed(list(revs))
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/revision.py", line 797, in iterate_revisions
    revisions, heads = fn(
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/revision.py", line 1452, in _collect_upgrade_revisions
    current_revisions = self.get_revisions(lower)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/revision.py", line 525, in get_revisions
    return sum([self.get_revisions(id_elem) for id_elem in id_], ())
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/revision.py", line 525, in <listcomp>
    return sum([self.get_revisions(id_elem) for id_elem in id_], ())
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/revision.py", line 550, in get_revisions
    return tuple(
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/revision.py", line 551, in <genexpr>
    self._revision_for_ident(rev_id, branch_label)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/revision.py", line 622, in _revision_for_ident
    raise ResolutionError(
alembic.script.revision.ResolutionError: No such revision or branch '5b0bd3b41a23'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/starlette/routing.py", line 677, in lifespan
    async with self.lifespan_context(app) as maybe_state:
  File "/home/admin-gpu/miniconda3/envs/detectron2/lib/python3.10/contextlib.py", line 199, in __aenter__
    return await anext(self.gen)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/server/api/server.py", line 518, in lifespan
    await run_migrations()
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/server/api/server.py", line 441, in run_migrations
    await db.create_db()
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/server/database/interface.py", line 56, in create_db
    await self.run_migrations_upgrade()
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/server/database/interface.py", line 64, in run_migrations_upgrade
    await run_sync_in_worker_thread(alembic_upgrade)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/utilities/asyncutils.py", line 91, in run_sync_in_worker_thread
    return await anyio.to_thread.run_sync(
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/anyio/to_thread.py", line 33, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 877, in run_sync_in_worker_thread
    return await future
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 807, in run
    result = context.run(func, *args)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/server/database/alembic_commands.py", line 24, in wrapper
    return fn(*args, **kwargs)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/server/database/alembic_commands.py", line 53, in alembic_upgrade
    alembic.command.upgrade(alembic_config(), revision, sql=dry_run)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/command.py", line 385, in upgrade
    script.run_env()
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/base.py", line 582, in run_env
    util.load_python_file(self.dir, "env.py")
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/util/pyfiles.py", line 94, in load_python_file
    module = load_module_py(module_id, path)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/util/pyfiles.py", line 110, in load_module_py
    spec.loader.exec_module(module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/server/database/migrations/env.py", line 174, in <module>
    apply_migrations()
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/utilities/asyncutils.py", line 243, in coroutine_wrapper
    return run_async_from_worker_thread(async_fn, *args, **kwargs)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/utilities/asyncutils.py", line 177, in run_async_from_worker_thread
    return anyio.from_thread.run(call)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/anyio/from_thread.py", line 47, in run
    return asynclib.run_async_from_thread(func, *args)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 906, in run_async_from_thread
    return f.result()
  File "/home/admin-gpu/miniconda3/envs/detectron2/lib/python3.10/concurrent/futures/_base.py", line 458, in result
    return self.__get_result()
  File "/home/admin-gpu/miniconda3/envs/detectron2/lib/python3.10/concurrent/futures/_base.py", line 403, in __get_result
    raise self._exception
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/server/database/migrations/env.py", line 168, in apply_migrations
    await connection.run_sync(do_run_migrations)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/sqlalchemy/ext/asyncio/engine.py", line 872, in run_sync
    return await greenlet_spawn(fn, self._proxied, *arg, **kw)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/sqlalchemy/util/_concurrency_py3k.py", line 192, in greenlet_spawn
    result = context.switch(value)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/server/database/migrations/env.py", line 136, in do_run_migrations
    context.run_migrations()
  File "<string>", line 8, in run_migrations
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/runtime/environment.py", line 928, in run_migrations
    self.get_context().run_migrations(**kw)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/runtime/migration.py", line 615, in run_migrations
    for step in self._migrations_fn(heads, self):
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/command.py", line 374, in upgrade
    return script._upgrade_revs(revision, rev)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/base.py", line 444, in _upgrade_revs
    with self._catch_revision_errors(
  File "/home/admin-gpu/miniconda3/envs/detectron2/lib/python3.10/contextlib.py", line 153, in __exit__
    self.gen.throw(typ, value, traceback)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/base.py", line 278, in _catch_revision_errors
    raise util.CommandError(resolution) from re
alembic.util.exc.CommandError: Can't locate revision identified by '5b0bd3b41a23'

Application startup failed. Exiting.
Server stopped!
```

I get the same error when I run any script that uses Prefect in its work.

Unfortunately, I don't know how to reproduce this error exactly because everything worked great when creating a virtual environment. As I mentioned earlier, I didn't change the dependencies in any way, didn't update or install anything. 

Please tell me what the problem may be and how to solve it ?

### Reproduction

```python3
prefect server start
```

### Error

```python3
___ ___ ___ ___ ___ ___ _____ 
| _ \ _ \ __| __| __/ __|_   _| 
|  _/   / _|| _|| _| (__  | |  
|_| |_|_\___|_| |___\___| |_|  

Configure Prefect to communicate with the server with:

    prefect config set PREFECT_API_URL=http://127.0.0.1:4200/api

View the API reference documentation at http://127.0.0.1:4200/docs

Check out the dashboard at http://127.0.0.1:4200



Traceback (most recent call last):
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/base.py", line 246, in _catch_revision_errors
    yield
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/base.py", line 456, in _upgrade_revs
    for script in reversed(list(revs))
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/revision.py", line 797, in iterate_revisions
    revisions, heads = fn(
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/revision.py", line 1452, in _collect_upgrade_revisions
    current_revisions = self.get_revisions(lower)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/revision.py", line 525, in get_revisions
    return sum([self.get_revisions(id_elem) for id_elem in id_], ())
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/revision.py", line 525, in <listcomp>
    return sum([self.get_revisions(id_elem) for id_elem in id_], ())
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/revision.py", line 550, in get_revisions
    return tuple(
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/revision.py", line 551, in <genexpr>
    self._revision_for_ident(rev_id, branch_label)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/revision.py", line 622, in _revision_for_ident
    raise ResolutionError(
alembic.script.revision.ResolutionError: No such revision or branch '5b0bd3b41a23'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/starlette/routing.py", line 677, in lifespan
    async with self.lifespan_context(app) as maybe_state:
  File "/home/admin-gpu/miniconda3/envs/detectron2/lib/python3.10/contextlib.py", line 199, in __aenter__
    return await anext(self.gen)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/server/api/server.py", line 518, in lifespan
    await run_migrations()
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/server/api/server.py", line 441, in run_migrations
    await db.create_db()
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/server/database/interface.py", line 56, in create_db
    await self.run_migrations_upgrade()
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/server/database/interface.py", line 64, in run_migrations_upgrade
    await run_sync_in_worker_thread(alembic_upgrade)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/utilities/asyncutils.py", line 91, in run_sync_in_worker_thread
    return await anyio.to_thread.run_sync(
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/anyio/to_thread.py", line 33, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 877, in run_sync_in_worker_thread
    return await future
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 807, in run
    result = context.run(func, *args)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/server/database/alembic_commands.py", line 24, in wrapper
    return fn(*args, **kwargs)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/server/database/alembic_commands.py", line 53, in alembic_upgrade
    alembic.command.upgrade(alembic_config(), revision, sql=dry_run)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/command.py", line 385, in upgrade
    script.run_env()
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/base.py", line 582, in run_env
    util.load_python_file(self.dir, "env.py")
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/util/pyfiles.py", line 94, in load_python_file
    module = load_module_py(module_id, path)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/util/pyfiles.py", line 110, in load_module_py
    spec.loader.exec_module(module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/server/database/migrations/env.py", line 174, in <module>
    apply_migrations()
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/utilities/asyncutils.py", line 243, in coroutine_wrapper
    return run_async_from_worker_thread(async_fn, *args, **kwargs)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/utilities/asyncutils.py", line 177, in run_async_from_worker_thread
    return anyio.from_thread.run(call)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/anyio/from_thread.py", line 47, in run
    return asynclib.run_async_from_thread(func, *args)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/anyio/_backends/_asyncio.py", line 906, in run_async_from_thread
    return f.result()
  File "/home/admin-gpu/miniconda3/envs/detectron2/lib/python3.10/concurrent/futures/_base.py", line 458, in result
    return self.__get_result()
  File "/home/admin-gpu/miniconda3/envs/detectron2/lib/python3.10/concurrent/futures/_base.py", line 403, in __get_result
    raise self._exception
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/server/database/migrations/env.py", line 168, in apply_migrations
    await connection.run_sync(do_run_migrations)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/sqlalchemy/ext/asyncio/engine.py", line 872, in run_sync
    return await greenlet_spawn(fn, self._proxied, *arg, **kw)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/sqlalchemy/util/_concurrency_py3k.py", line 192, in greenlet_spawn
    result = context.switch(value)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/prefect/server/database/migrations/env.py", line 136, in do_run_migrations
    context.run_migrations()
  File "<string>", line 8, in run_migrations
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/runtime/environment.py", line 928, in run_migrations
    self.get_context().run_migrations(**kw)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/runtime/migration.py", line 615, in run_migrations
    for step in self._migrations_fn(heads, self):
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/command.py", line 374, in upgrade
    return script._upgrade_revs(revision, rev)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/base.py", line 444, in _upgrade_revs
    with self._catch_revision_errors(
  File "/home/admin-gpu/miniconda3/envs/detectron2/lib/python3.10/contextlib.py", line 153, in __exit__
    self.gen.throw(typ, value, traceback)
  File "/home/admin-gpu/.cache/pypoetry/virtualenvs/sync-fihes-videos-DL3CyE40-py3.10/lib/python3.10/site-packages/alembic/script/base.py", line 278, in _catch_revision_errors
    raise util.CommandError(resolution) from re
alembic.util.exc.CommandError: Can't locate revision identified by '5b0bd3b41a23'

Application startup failed. Exiting.
Server stopped!
```

### Versions

```Text
Version:             2.10.21
API version:         0.8.4
Python version:      3.10.11
Git commit:          da816542
Built:               Thu, Jul 13, 2023 2:05 PM
OS/Arch:             linux/x86_64
Profile:             default
Server type:         ephemeral
Server:
  Database:          sqlite
  SQLite version:    3.41.2
```

**Comment by urimandujano** [comment]
I've seen this happen before when the DB is initialized using one version of `prefect server start` and then another version (usually older) of `prefect server start` starts up and uses the same DB.

If you don't have need for persisted data, you can try removing the prefect DB at `$HOME/.prefect/prefect.db` before starting the server. Unfortunately virtual environments won't help isolate these DBs from each other.

**Comment by medphisiker** [comment]
Thank you for your help. The database of Prefect was not critical, I deleted it and everything worked again. Yes, I have two virtual environments with slightly different versions of Prefect. Then I'll fix the same one )

**Comment by figuernd** [comment]
> I've encountered an issue that mirrors what's been discussed, but with a notable difference: I'm not utilizing multiple environments. My transition from Prefect Cloud to Ephemeral mode introduced this problem, and currently, it persists in both Cloud and Ephemeral modes.

Same, I have not switched versions and this error began after a server shutdown; guessing that interrupted a process that resulted in a corrupt state.

I'll probably have to delete the database but that is not really a solution.

**Comment by tamboles98** [comment]
Hi. I faced this issue with a Prefect server we were running together with our own postgres database.

The cause, as @figuernd mentioned, turned out to be that someone deployed a newer version of Prefect: 2.16 vs the usual 2.14. We managed to get everything working again by having a Prefect 2.16 installation configured to connect to the postgres database, then running `prefect server database downgrade 6b63c51c31b4` with it.

Version id `6b63c51c31b4` comes from the last revision file in `src/prefect/server/database/migrations/versions/postgresql` for the old version you want to redeploy, in this case for 2.14.21: https://github.com/PrefectHQ/prefect/blob/9a059bfeb5ed688c334264d61286a663e9805058/src/prefect/server/database/migrations/versions/postgresql/2024_01_05_101034_6b63c51c31b4_add_sender_to_flowruninput.py
